### PR TITLE
ETCbits

### PR DESCRIPTION
**Pull request by alina-krt**
1st 3D collection and only 2nd NFT collection on Ethereum Classic

### CODE REVIEW

File: content/news/media/media.collection.en.yaml [hunk 1]
```diff
@@ -1,4 +1,10 @@
--  
+  date: 2021-11-24
+  link: https://medium.com/@persn.assist/etcbits-1st-3d-collection-and-only-2nd-nft-collection-on-ethereum-classic-547e787f5633
+  author: ETCBits
+  source: Medium
+  title: "ETCbits — 1st 3D collection and only 2nd NFT collection on Ethereum Classic"
+  tags: ["announcement", "development", "nft”]
```
Comment (alina-krt) [inline content/news/media/media.collection.en.yaml:6]: tags: ["announcement", "development", "nft"]

Comment (alina-krt) [inline content/news/media/media.collection.en.yaml:6]: tags: ["announcement", "development", "nft"]

**Review by IstoraMandiri — APPROVED** [review]
Looks good, I changed the tag "nft" to "application", as we don't have an NFT tag at the moment.

**Comment by IstoraMandiri** [comment]
Someone in the discord raised a concern about this project being legit or not, I haven't checked it out but I've converted this to a draft for now so we can get more feedback on the project before merging.

**Comment by alina-krt** [comment]
@IstoraMandiri 
how the limiting of a feed to ETC specifically related content works?
it won't be on the website? 
The project https://etcpunks.com/marketplace doesn't have this problems?

**Comment by alina-krt** [comment]
@IstoraMandiri 
contract address 
https://blockscout.com/etc/mainnet/address/0x4938B060481D7aeD59fcD21abd51f29B48994EC8/contracts

**Comment by IstoraMandiri** [comment]
> how the limiting of a feed to ETC specifically related content works?

I think we should still list your project on the website. The concern is that the news on the front page was not really for third party projects, of which there were none previously, and going forward, should be reserved for only for ETC News.

I'll follow up with another PR that removes all third party apps from the front page to keep things fair.

**Comment by alina-krt** [comment]
@IstoraMandiri 
Thank you for your support!)
isn't it still the ETC related topics?
mostly networks do support those who develop for the network and tell about what's new and what's going on 🤔
In this case people don't have to look for all the updates all around the media sources but find related info on the website.
As a plus for now it's a wave of NFT projects and there are only two on  ETC.
What else can we do to get a support from you?)
Maybe mentioning in your medium account?

**Comment by IstoraMandiri** [comment]
If you would like to suggest some alternatives, please continue discussion in the other PR #693, whereby ETCBits is included on the folowing pages:

https://deploy-preview-693--ethereumclassic.netlify.app/news/
https://deploy-preview-693--ethereumclassic.netlify.app/ecosystem/apps-and-protocols/

**Comment by alina-krt** [comment]
@IstoraMandiri thank you very much for your support 🙂

**Review by gitr0n1n — APPROVED** [review]
LGTM!

**Comment by gitr0n1n** [comment]
> > how the limiting of a feed to ETC specifically related content works?
> 
> I think we should still list your project on the website. The concern is that the news on the front page was not really for third party projects, of which there were none previously, and going forward, should be reserved for only for ETC News.
> 
> I'll follow up with another PR that removes all third party apps from the front page to keep things fair.

Makes sense. Thanks for working through this.

**Pull request closed without merging**